packages/clay-icon/src/ClayIcon.soy

```diff
@@ -32,8 +44,12 @@
 		{/if}
 	{/let}
 
-	<svg {$attributes}>
-		<title>{$title ?: $id ?: $symbol}</title>
-		<use xlink:href="{$spritemap}#{$symbol}" />
-	</svg>
+	<span {$containerAttributes}>
+		<svg {$attributes}>
+			{if $title}
```

Comment:
If we don't need title inside `svg` for accessibility anymore I'd remove it.

Reply:
Wouldn't removing the title break the api for those that are already using ClayIcon?

Reply:
I don't mean removing title from API but from inside the svg.

```
<svg {$attributes}>
	<use xlink:href="{$spritemap}#{$symbol}" />
</svg>
```